pyproject.toml

```
@@ -192,3 +196,19 @@ strict = true
 [tool.codespell]
 skip = ".git,*.pdf,*.svg"
 ignore-words-list = "theis,coo,homogenous"
+
+[tool.towncrier]
+package = "anndata"
+directory = "docs/release-notes"
+filename = "docs/release-notes/{version}.md"
+single_file = false
+package_dir = "src"
+issue_format = "[#{issue}](https://github.com/scverse/anndata/pull/{issue})"
+title_format = "({version})=\n### {version} `{project_date}`"
```

After merging #1346, I found that these don’t actually work: seems like MyST doesn’t support anchors that start with a number.

So in https://github.com/scverse/scanpy/pull/3172, I started prefixing them with a `v` like this to get anchors:

```suggestion
title_format = "(v{version})=\n### {version} `{project_date}`"
```